### PR TITLE
Vendoring

### PR DESCRIPTION
**Pull request by grobie**
Use govendor to vendor all dependencies and shield against changing APIs
or repository unavailability.

### CODE REVIEW

File: Makefile.COMMON [hunk 1]
```diff
@@ -44,7 +44,7 @@ SRC    ?= $(shell find . -type f -name "*.go" ! -path "./.build/*")
 GOOS   ?= $(shell uname | tr A-Z a-z)
 GOARCH ?= $(subst x86_64,amd64,$(patsubst i%86,386,$(shell uname -m)))
 
-GO_VERSION ?= 1.5.3
+GO_VERSION ?= 1.6
```
Comment (brian-brazil) [inline Makefile.COMMON]: Go 1.6 has known major bugs, we shouldn't upgrade anything to it yet

Comment (grobie) [inline Makefile.COMMON]: Removed.

**Comment by grobie** [comment]
Wait one second please, I'm validating all available license files are included.

**Comment by grobie** [comment]
Ready for review.

**Comment by brian-brazil** [comment]
:+1:

**Pull request merged**